javascript/javascript1/week1/homework.js

```diff
@@ -0,0 +1,91 @@
+// Step 2
+// My freecodecamp username - @Natata08
+
+//Step 3
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1986; //number
+const yearFuture = 2025; //number
+const age = yearFuture - yearOfBirth;
+const messageAboutFututeAge = 'You will be ' + age + ' years old in ' + yearFuture;
+
+console.log(messageAboutFututeAge);
+
+// Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = false; //boolean
+let messageAboutDogAge;
+
+if (shouldShowResultInDogYears) {
+  messageAboutDogAge = 'Your dog will be ' + dogYear * 7 + ' dog years old in ' + dogYearFuture;
+} else {
+  messageAboutDogAge = 'Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture;
+}
```

Comment:
Very nice. In line 23 and line 25 you are repeating yourself quite a lot. Most of these two lines are the same. Can you rewrite this, such that only the `dogYear` is changed based on the condition, and the string concatenation is done outside of the if-else clause.


See https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Done! It really good concept, thank you. 
I hope I fixed it correctly. I've found out about a ternary operator and replaced all if-else with it.